common/coin_test.go

```diff
@@ -0,0 +1,43 @@
+package common_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/zeta-chain/zetacore/common"
+)
+
+func Test_GetAzetaDecFromAmountInZeta(t *testing.T) {
+	tt := []struct {
+		name       string
+		zetaAmount string
+	}{
```

Comment:
Can we add a test: non valid zetaAmount => error

Reply:
27c727ade9f75113e34bc452b06286385f41948c